### ISSUE TITLE
[Bug] MangaPark: Crashes on author parse

### ISSUE THREAD

**Issue by jobobby04** [inorichi/tachiyomi-extensions#4365]
## Device information
- Tachiyomi version: TachiyomiSY (Mangapark is not delegated) (Also tested on stable)
- Android version: 6

## Source information
- Name: Mangapark
- Extension version: 1.2.17

## Steps to reproduce
1. View a manga from the source
2. It crashes

### Expected behavior
Manga should parse

### Actual behavior
It crashes with this error
![WvQG3Xs 1](https://user-images.githubusercontent.com/17078382/93043189-167e0c80-f61f-11ea-9b66-ec23a59987b5.png)

## Other details
Not my fault, I do nothing with mangapark! This issue is on stable as well!

**Comment by CarlosEsco** [comment]
Is the a byproduct of updating kotlin in extensions?

**Comment by arkon** [comment]
Yup.

Full stacktrace from a 5.0 emulator using 0.10.4:

```
09-14 10:28:04.064 3915-3957/eu.kanade.tachiyomi E/ACRA: ACRA caught a IllegalStateException for eu.kanade.tachiyomi
    java.lang.IllegalStateException: Fatal Exception thrown on Scheduler.Worker thread.
        at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:4)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
        at java.lang.Thread.run(Thread.java:818)
     Caused by: java.lang.NoSuchMethodError: No direct method <init>(ILjava/lang/Class;Ljava/lang/String;Ljava/lang/String;I)V in class Lkotlin/jvm/internal/FunctionReferenceImpl; or its super classes (declaration of 'kotlin.jvm.internal.FunctionReferenceImpl' appears in /data/app/eu.kanade.tachiyomi-1/base.apk:classes2.dex)
        at eu.kanade.tachiyomi.extension.en.mangapark.MangaPark$mangaDetailsParse$1$2$1.<init>(MangaPark.kt)
        at eu.kanade.tachiyomi.extension.en.mangapark.MangaPark$mangaDetailsParse$1$2$1.<clinit>(MangaPark.kt)
        at eu.kanade.tachiyomi.extension.en.mangapark.MangaPark.mangaDetailsParse(MangaPark.kt:99)
        at eu.kanade.tachiyomi.source.online.ParsedHttpSource.mangaDetailsParse(ParsedHttpSource.kt:1)
        at eu.kanade.tachiyomi.source.online.HttpSource$fetchMangaDetails$1.call(HttpSource.kt:2)
        at eu.kanade.tachiyomi.source.online.HttpSource$fetchMangaDetails$1.call(HttpSource.kt:1)
        at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:1)
        at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:3)
        at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1$requestArbiter$1.request(OkHttpExtensions.kt:4)
        at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber$1.request(OperatorSubscribeOn.java:3)
        at rx.Subscriber.setProducer(Subscriber.java:8)
        at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.setProducer(OperatorSubscribeOn.java:1)
        at rx.internal.operators.OnSubscribeMap$MapSubscriber.setProducer(OnSubscribeMap.java:1)
        at rx.Subscriber.setProducer(Subscriber.java:6)
        at rx.internal.operators.OnSubscribeMap$MapSubscriber.setProducer(OnSubscribeMap.java:1)
        at rx.Subscriber.setProducer(Subscriber.java:6)
        at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:5)
        at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:1)
        at rx.Observable.unsafeSubscribe(Observable.java:2)
        at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:2)
        at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:1)
        at rx.Observable.unsafeSubscribe(Observable.java:2)
        at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:4)
        at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:1)
        at rx.Observable.unsafeSubscribe(Observable.java:2)
        at rx.internal.operators.OnSubscribeDefer.call(OnSubscribeDefer.java:3)
        at rx.internal.operators.OnSubscribeDefer.call(OnSubscribeDefer.java:1)
        at rx.Observable.unsafeSubscribe(Observable.java:2)
        at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:4)
        at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:1)
        at rx.Observable.unsafeSubscribe(Observable.java:2)
        at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:4)
        at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:2)
        at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:2)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:237) 
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152) 
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
        at java.lang.Thread.run(Thread.java:818) 
```

Works fine with the preview app. I'll just do a new stable release.

**Comment by arkon** [comment]
This should be fixed by updating to Tachiyomi v0.10.5.
